src/OpenTelemetry/Metrics/Processors/MetricConsoleExporter.cs

```diff
@@ -34,12 +34,45 @@ public override void OnEnd(MetricItem data)
             {
                 var tags = metric.Attributes.ToArray().Select(k => $"{k.Key}={k.Value?.ToString()}");
 
+                string valueDisplay = string.Empty;
+                if (metric is ISumMetric sumMetric)
+                {
+                    if (sumMetric.Sum is double doubleSum)
+                    {
+                        valueDisplay = ((double)doubleSum).ToString();
```

Comment:
InvariantCulture?

Reply:
I think we missed a few more .ToString() calls to add `InvariantCulture`